src/utils/stringUtils.ts

```diff
@@ -1,6 +1,7 @@
 import pickBy from 'lodash.pickby';
+
 export const storForbokstav = (tekst: string) => {
-    return tekst ? tekst.toLowerCase().replace(/\b\w/, (v) => v.toUpperCase()) : tekst;
+    return tekst ? tekst.replace(/\b\w/, (v) => v.toUpperCase()) : tekst;
```

Comment:
Denne endringen gjør at funksjonen ikke fungerer på ord med bare store bokstaver.
Eks. ble det brukt på Rolle slik:
`storForbokstav("VEILEDER") = "Veileder"`
Det fungerer ikke med den endringen der.

Reply:
Jeg føler at funksjonsbeskrivelsen her ikke indikerer at hele resten av teksten vil bli lower-caset samtidig, så det var motivasjonen for å endre denne.

Har prøvd å se etter om noe blir rart som følge av endringen, kan dobbeltsjekke det

Reply:
Jeg laget en ny funksjon "kunStorForbokstav"! 🤣